bin/inx-chronicle/config.template.toml

```diff
@@ -12,4 +12,5 @@ port = 9092
 allow_origins = ["0.0.0.0"]
 
 [syncer]
-max_simultaneous_requests = 10
+earliest_milestone = 0
+rate_limit = "100ms"
```

Comment:
As I said I am not sure about this because this is completely arbitrary and cannot "react" to how solidification takes on the system. There will be huuuge differences for solidfying milstones that have only 10 messages in them and those that might have 1000. So sending a new request every 100ms can become problematic.